code/modules/paperwork/clipboard.dm

```diff
@@ -1,154 +1,155 @@
+#define PAPERWORK	1
+#define PHOTO		2
+#define IS_PEN(x)	(istype(x, /obj/item/weapon/pen))
+
 /obj/item/weapon/clipboard
 	name = "clipboard"
+	desc = "It looks like you're writing a letter. Want some help?"
 	icon = 'icons/obj/bureaucracy.dmi'
 	icon_state = "clipboard"
 	item_state = "clipboard"
-	throwforce = 0
 	w_class = WEIGHT_CLASS_SMALL
-	throw_speed = 3
-	throw_range = 10
```

Comment:
the throw_range should be reduced to `7`, for sure, but otherwise, same question as @Crazylemon64